content/en/docs/tutorials/security/apparmor.md

```diff
@@ -132,7 +132,7 @@ discussed in [Setting up nodes with profiles](#setting-up-nodes-with-profiles).
 
 ```shell
 # This example assumes that node names match host names, and are reachable via SSH.
-NODES=($(kubectl get nodes -o name))
+NODES=($(kubectl get nodes -o name | cut -d/ -f2))
```

Comment:
How about this:
```suggestion
NODES=($( kubectl get node -o jsonpath='{.items[].status.addresses[?(.type == "Hostname")].address}' ))
```

Aside: we should also list bash as a prerequisite; this syntax is a bash extension to POSIX.